### DISCUSSION TITLE
Not going to the next step

### DISCUSSION THREAD

**Discussion by ngugikubai** [skills/.github#515] [Communicate using Markdown]
I've followed all the steps for the first markdown but I'm not able to move to the next step after commiting changes and refreshing the page. Kindly assist

**Comment by vinayadasepalli** [comment]
It seems the content needs updating and the steps need to be revised to reflect the latest software/website changes.

**Reply by ngugikubai** [reply]
Thanks for this! I started the activity again and it worked

**Comment by yimengzhiyan** [comment]
maybe you should go to the action page to check if anything is go wrong。

**Reply by ngugikubai** [reply]
Thanks for this! I started the activity again and it worked!
